loading.bs

```diff
@@ -1558,6 +1561,22 @@ A [=request=] |browserRequest| <dfn lt="matches the stored exchange|doesn't matc
 1. If |browserRequest|'s [=request/url=] is not [=url/equal=] to
     |storedExchange|'s [=exchange/request URL=], return "mismatch".
 
+1. If |browserRequest|'s [=request/header list=] contains a [=header=]
+    [=header/named=] \``Cookie`\` and |storedExchange|'s [=exchange/response=]'s
+    [=response/header list=] contains a [=header=] [=header/named=] \``Vary`\`,
+    then:
+
+    1. Let |fieldValues| be the [=list=] containing the elements corresponding to
+        the [=http/field-names=] in the [=Vary=] header field value.
```

Comment:
Does

```suggestion
    1. Let |fieldValues| be the [=list=] of [=http/field-names=] in the [=Vary=] header field value.
```

work? Or "a new [=list=] whose elements are the [=http/field-names=] …"?

Reply:
Done.